changelog

```diff
@@ -106,6 +145,12 @@ Version 0.9.9:
       * Added more fancy passenger ships in the Paradise systems. (@Pointedstick)
       * The cloaking device is now unplunderable. (@Bladewood)
       * Switched some landscape images so that fewer of them are used for multiple planets. (@Amazinite)
+      * All red wormholes now require the quantum keystone attribute to be used. (@Nescio0)
+      * Wasps will now spawn in small Core pirate fleets. (@Nescio0)
+      * Test dummy ships now only aim to disable instead of destroying. (@tehhowch, @Amazinite)
+      * Command centers are now unplunderable. (@AlexBassett)
+      * Navy ships, human medium warships, and human heavy warships now come equipped with hand to hand weapons. (@Fzzr)
```

Comment:
Update after #4216: 
`* Navy Ships, human medium warships, human heavy warships, human heavy freighters, and certain human variants now come equipped with hand to hand weapons. (@Fzzr)`